src/js/bellows.js

```diff
@@ -52,6 +52,12 @@
             this._bindEvents();
         },
 
+        destroy: function() {
+            this.$bellows
+                .removeData(this.name)
+                .off(this.options.event);
```

Comment:
It was originally exposed as an option in order to override the behaviour of Tappy, which we had included by default in bellows when it was first being developed. We later took tappy out in favour of people implementing a fast tap solution themselves.